bin/rancher_image_chart_promote.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+set -e
+
+. libstd-ecm.sh
+
+usage() {
+    echo "usage: $0 [h] <tag> <stable_or_latest>
+    -h    show help
+    -d    dry run mode
+
+examples:
+    $0 v2.7.1 v2.8.3
+    $0 -d"
+}
+
+while getopts 'hd' c; do
+    case $c in
+    h)
+        usage
+        exit 0
+    ;;
+    d)
+        DRY_RUN=true
+    ;;
+    *)
+        usage
+        exit 1
+    ;;
+    esac
+done
+
+has_drone
+
+if [ $# -ne 2 ]; then
```

Comment:
I reach this condition if I run `./rancher_image_chart_promote.sh -d v2.7.1 latest`. What's the right command for a dry run?

Reply:
I've removed dry run for now as it's not really behaving as I'd expect to. I'll add it back when this code gets used. The active code is in the rancher repo.